pkg/terraform/build.go

```diff
@@ -13,7 +15,33 @@ COPY . $BUNDLE_DIR
 RUN cd %s && terraform init -backend=false
 `
 
+// BuildInput represents stdin passed to the mixin for the build command.
+type BuildInput struct {
+	Config MixinConfig
+}
+
+// MixinConfig represents configuration that can be set on the terraform mixin in porter.yaml
+// mixins:
+// - terraform:
+//	  version: 0.12.17
+type MixinConfig struct {
+	Version string `yaml:"version,omitempty"`
```

Comment:
I agree with Vaughn that we are reserving "version", so another name would be better. The other mixins use `clientVersion`.